Sources/Rules/WrapFunctionBodies.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  WrapFunctionBodies.swift
+//  SwiftFormat
+//
+//  Created by Manuel Lopez on 12/15/24.
+//  Copyright © 2024 Nick Lockwood. All rights reserved.
+//
+
+import Foundation
+
+public extension FormatRule {
+    /// Wrap single-line function and init bodies onto multiple lines.
+    static let wrapFunctionBodies = FormatRule(
+        help: "Wrap single-line function and init bodies onto multiple lines.",
+        sharedOptions: ["linebreaks", "indent"]
+    ) { formatter in
+        formatter.forEach(.keyword) { keywordIndex, keyword in
+            guard ["func", "init"].contains(keyword.string),
```

Comment:
We can include subscripts too right?

```suggestion
            guard ["func", "subscript", "init"].contains(keyword.string),
```

Reply:
Adding it back in. Initially took it out because the rule name is now misleading but I'm okay either way.